playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/upgrade.yml

```diff
@@ -16,14 +45,20 @@
 ###############################################################################
 - name: Upgrade master
   hosts: oo_masters_to_config
+  vars:
+    openshift_master_ha: "{{ groups.oo_masters_to_config | length > 1 }}"
```

Comment:
@abutcher did we introduce a fact for this? Or am I thinking about a pending change?